executable/FnmApp.re

```diff
@@ -3,6 +3,7 @@ let version = Fnm.Fnm__Package.version;
 module Commands = {
   let use = (version, quiet) => Lwt_main.run(Use.run(~version, ~quiet));
   let alias = (version, name) => Lwt_main.run(Alias.run(~name, ~version));
+  let default = (version, name) => Lwt_main.run(Alias.run(~name, ~version));
```

Comment:
The shorter way would've been to set `~name="default"` but that doesn't look or feel right.

Reply:
I totally understand your take on this, but considering how hard is it for a new contributor (and new Reason user) to understand Cmdliner, I think it's better to set the `default` magic string here and not in the Cmdliner bloat. This module (`Commands`) is easy to read and pretty much sums everything up.

And it's not that we don't have the magic `default` string here, it is just written in the weird land of Cmdliner, where it's harder to follow on things 😜 

The other approach was to remove the `default` function completely because it is exactly like the `alias command`, and use the `const("default")` you have used into `Commands.alias`.

However, I think your choice of making a new command looks much nicer. Every command exposes a function in the `Commands` module. Cmdliner is just something we have to use in order to make good CLIs, but it's just an adapter 😄 

What do you think?

Reply:
After much thinking and mulling in what you said, I agree that the magic string in the `Commands` module is very suitable. 

But having each `Commands` expose a new command with its own function is really intuitive. It also means we can customize CLI helper comments and whatnot for each command without having to overlap, say, `alias` and `default`. 

:D This is fun.